extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -115,6 +116,7 @@ export const StateProperty = {
   SUPPORTED_BROWSER_STATE: 'supportedbrowserstate',
   STORY_HAS_AUDIO_STATE: 'storyaudiostate',
   UI_STATE: 'uistate',
+  UI_VISIBILITY_STATE: 'uivisibilitystate',
```

Comment:
Two nits about the name:

1. We should probably be clear that this is the system layer UI.
2. A value of `true` doesn't make me feel like I understand that this is visible.

Maybe something like `SYSTEM_LAYER_UI_IS_VISIBLE_STATE`?  I know that doesn't sound great either, feel free to wordsmith

Reply:
As we might add some new UI elements, like the swipe up interaction, I was trying to keep it more generic and avoid adding system layer in the name.
What do you think of these: `STORY_UI_IS_VISIBLE_STATE`, `UI_IS_VISIBLE_STATE`, `SYSTEM_UI_IS_VISIBLE_STATE`

Reply:
Thanks for the suggestion! I went with `SYSTEM_UI_IS_VISIBLE_STATE`